ropetest/doatest.py

```diff
@@ -0,0 +1,85 @@
+import base64
+import hashlib
+import hmac
+import multiprocessing
+try:
+    import pickle
+except ImportError:
```

Comment:
other way around. pickle will always succeed, it's `cPickle` that might not depending on Python version / implementation.

So it should be:

```python
try:
    import cPickle as pickle
except ImportError:
    import pickle
```

Reply:
Huh, I copied this from elsewhere without noticing. Yeah, that's not right!